### ISSUE TITLE
Publics

### ISSUE THREAD

**Issue by JE33-555** [KyleTheScientist/Bark#14]
If I use bark in a modded. Open the menu and the disable my internet. When I enable it again I still have the menu open just not in a room. Meaning I can go into a non modded room with bark.

**Comment by camera12345** [comment]
Ye dont do that

**Comment by dedouwe26** [comment]
I think it's a problem of Utilla

**Comment by developer9998** [comment]
From the looks of it, this is still an issue.. Yikes

**Comment by dedouwe26** [comment]
Yes, but the mod is thinking its in a modded lobby and Utilla handles the lobbies
I think.

**Comment by AmongusR** [comment]
i think its tmploader that deals with modded lobbies becuz when i didn't have tmp loader, the buttons wouldn't work and i wasn't able to go into modded lobbies

**Comment by bubzilla** [comment]
> i think its tmploader that deals with modded lobbies becuz when i didn't have tmp loader, the buttons wouldn't work and i wasn't able to go into modded lobbies

no Utilla does it. TMPLoader just loads text.
